CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to Amadeus Open Source Manifesto
+Thank you for considering donating your precious time! We welcome contributions and appreciate your effort in improving our Manifesto. To ensure a smooth contribution process, please follow the guidelines below.
+
+## Who can contribute
+This Manifest serves as a contract between our organization, Amadeus, and the Open Source community. Therefore, any significant changes must originate from our employees. However, technical modifications — such as correcting typos or adjusting the document's layout—can also be proposed by external contributors.
+
+## How to contribute
+We accept the suggestions in the form of an **Issue** or a **Pull request**. Alternatively, you can use our internal Teams Channel: **OpenSource@1A** or by sending the suggestion to *opensource@amadeus.com.*
```

Comment:
Small nit: for the mail address at the end, don't include the final dot in the emphasis: 

```
...the suggestion to *opensource@amadeus.com*.
```